packages/git-semver-tags/index.js

```diff
@@ -13,36 +14,38 @@ function lernaTag (tag, pkg) {
   }
 }
 
-module.exports = function (callback, opts) {
-  opts = opts || {}
+module.exports = function gitSemverTags (opts, callback) {
+  if (typeof opts === 'function') {
+    callback = opts
+    opts = {}
+  }
+  var options = Object.assign({ maxBuffer: Infinity, cwd: proc.cwd() }, opts)
```

Comment:
```javascript
const options = { maxBuffer: Infinity, cwd: proc.cwd(), ...opts}
```

We can use object spread now that the minimum Node version we support is Node 8 LTS, assuming we don't need to rely on the behavior of `Object.assign` to call setters (also, I changed `var` to `const`)

Reply:
cool cool. I should read about the diffs again, any suggestions? I started using it everywhere and still prefer it over spread for some unknown reason. ;d

Reply:
> I should read about the diffs again, any suggestions?

I tend to just read [Mozilla documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax). :sweat_smile:  In my experience their examples are pretty good.

> I started using it everywhere and still prefer it over spread for some unknown reason. ;d

That's totally fine. My suggestion, after further consideration, was purely based on  aesthetics (from my perspective).